source/manual/howto-change-a-government-following-an-election.html.md

```diff
@@ -0,0 +1,52 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Change a government following a general election
+section: Publishing
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-07-12
+review_in: 6 months
+---
+
+### Change a government in Whitehall
+
+Following a general election or other reason for governmental change the current Government in Whitehall will need to be closed, and a new one created.
+
+#### Closing a government
+
+Government's are listed at the [government path of Whitehall Admin](https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/governments).
+
+1. Select the current government, and click the `Prepare to close this government` link.
+2. Close the government.
+
+This will close the current government and remove all [ministerial appointments](https://www.integration.publishing.service.gov.uk/government/ministers). Changes will instantly appear on the live site.
+
+The Content Support team will be able to re-assign ministers once the new government has been opened.
+
+#### Open a new government
+
+[Create a new government](https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/governments/new)
+
+This will instantly create a new government. Changes can be seen in the Rails console by running `Government.current`.
+
+The new government should also instantly appear in the response from [/api/governments](https://www.integration.publishing.service.gov.uk/api/governments).
+
+#### Republishing Political Content
+
+Political Content is encapsulated by Whitehall's [PoliticalContentIdentifier](https://github.com/alphagov/whitehall/blob/master/lib/political_content_identifier.rb). You can identify a piece of political content by calling `PoliticalContentIdentifier.political?(edition)`.
```

Comment:
I don't think this is strictly true.

As I understand each edition has a flag of `political` which determines whether it will be flagged as historic or not. Only certain users can set this flag: managing editors, gds editor, gds admins. That flag is set by default using the `PoliticalContentIdentifier` class.

So a better way to think of it is that `PoliticalContentIdentifier` is used to preset whether an edition is political or not, this can be overwrote by a Managing Editor or a GDS Editor. The check for whether content is in history mode is: `edition.political? && !edition.government.current?`

Reply:
:+1: yep, this makes sense. I'll adjust the docs.